cmd/estuary-shuttle/main.go

```diff
@@ -1337,10 +1337,11 @@ func (s *Shuttle) handleAdd(c echo.Context, u *User) error {
 	}
 
 	return c.JSON(http.StatusOK, &util.ContentAddResponse{
-		Cid:          nd.Cid().String(),
-		RetrievalURL: util.CreateRetrievalURL(nd.Cid().String()),
-		EstuaryId:    contid,
-		Providers:    s.addrsForShuttle(),
+		Cid:                 nd.Cid().String(),
+		DwebRetrievalURL:    util.CreateDwebRetrievalURL(nd.Cid().String()),
+		EstuaryRetrievalURL: util.CreateEstuaryRetrievalURL(nd.Cid().String()),
```

Comment:
this is a breaking change for some users. best to leave it as `RetrievalURL` (which is fine)

Reply:
Oh yeah. Please put back the `RetrievalURL` but leave the `EstuaryRetrievalURL` as is.